lib/puppet/jenkins.rb

```diff
@@ -4,7 +4,11 @@ module Jenkins
     # @return [String] Full path to the Jenkins user's home directory
     def self.home_dir
       begin
-        return File.expand_path('~jenkins')
+        if Facter.value(:osfamily) == 'OpenBSD'
+          return File.expand_path('~_jenkins')
```

Comment:
yes, more or less every port that has a daemon that not necessarily must run as root, has such users/group names, see: http://cvsweb.openbsd.org/cgi-bin/cvsweb/~checkout~/ports/infrastructure/db/user.list?rev=1.249&content-type=text/plain